### PR TITLE
Contributors documentation

### PR DESCRIPTION
**Pull request by whitetigle**
Following the [issue](https://github.com/MangelMaxime/Fulma/issues/28) opened by @evilz. Here's a first draft for a contributor's guide to Fulma project

It WILL have errors and it WILL be incomplete. So if you have some time, please review it so that I can update it accordingly.

Thanks!

(just don't take car of the branch name, although it will hint what I am up to 😉 )

### CODE REVIEW

**Comment by evilz** [comment]
Nice 👍

**Comment by MangelMaxime** [comment]
Thank you 👍

File: GUIDELINES.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+## Install dependencies and first build
+
+In order to install the dependencies, generate the docs and build the project just type the following commands:
+
+On windows:
+```shell
+./build.cmd
+```
+
+On Unix:
```
Comment (whitetigle) [inline GUIDELINES.md]: Just a comment regarding this change: 
I would not say MacOS-Whatever is Unix nor would I ever tell anybody that Linux is Unix.
I think people may get hanged in some countries just for telling such crazy things 😉 
But hey, I understand it's funny to summon some ancient trolls sometimes :trollface:

Comment (MangelMaxime) [reply GUIDELINES.md]: Stop playing with the name, I already have Thot name problem to fix somehow...

**Pull request merged**